docs/optimization_guide/nncf/introduction.md

```diff
@@ -0,0 +1,47 @@
+# Optimizing Models at Training Time {#tmo_introduction}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   
+   qat_introduction
+   filter_pruning
+
+@endsphinxdirective
+
+## Introduction
+Training-time model optimization is a way to get a more efficient and HW-friendly model when applying optimization methods with fine-tuning. It can help when the [post-training optimization](@ref pot_introduction) does not provide the desired accuracy or performance results. OpenVINO&trade; does not have training capabilities but it provides a Neural Network Compression Framework (NNCF) tool that can be used to integrate training-time optimizations supported by OpenVINO in the training scripts created using 3rd party frameworks, such as PyTorch or TensorFlow 2. 
+
+To apply training-time optimization methods you need:
+- A floating-point model in the framework representation.
+- Training script written with framework API.
+- Training and validation datasets.
+
+Figure below shows a common workflow of applying training-time optimizations with NNCF.
+
+![](../../img/nncf_workflow.png)
+
+## Optimization options
+There are two methods available to improve model performance with OpenVINO&trade;:
+- **Quantization-aware Training (QAT)** is a **recommended** method that provides fast and accurate results. Currently, a HW-compatible (CPU, GPU, VPU) QAT for 8-bit inference is available. For details, see [Quantization-aware Training](./qat.md) documentation.
+- **Filter Pruning** is used to remove unnecessary or redundant filters from Convolutional Neural Networks. It is usually not used stand-alone but can be stacked with QAT to get additional speedup on top of it. This method is resource-demanding and requires model retraining which can take a significant amount of time in the general case. Thus, we recommend trying it when 8-bit quantization does not provide enough performance improvement. For details, see [Filter Pruning](./filter_pruning.md) documentation.
+
+## Installation
+NNCF is open-sourced on [GitHub](https://github.com/openvinotoolkit/nncf) and distributed as a separate package. It is also available on PyPI. We recommend installing it to the Python* environment where the framework is installed.
+
```

Comment:
The we should mention ONNX and probably IR in the future which will sound wordy )

Reply:
I would think about whether should we guide the user to install NNCF into the same environment or create a separate one? The separate could be preferable since the right way is to create a separate script for model optimization (based on the original training script, however).

Reply:
That's a good question. As you mentioned, the separate env is preferable, but to be honest, I often install NNCF into the existing env to get optimization capabilities.